internal/test/krb/root/usr/local/bin/init.sh

```diff
@@ -25,6 +25,18 @@ kadmin.local -q "addprinc -pw ${KERBEROS_PASSWORD} ${KERBEROS_USERNAME}"
 echo -n "" >/etc/krb5kdc/kadm5.acl
 echo "${KERBEROS_USERNAME}@TESTING.CONTAINERSSH.IO *" >>/etc/krb5kdc/kadm5.acl
 
+echo -e "\e[32mAdding host principal testing.containerssh.io ...\e[0m"
+kadmin.local -q "addprinc -randkey host/testing.containerssh.io"
```

Comment:
Ideally, we wouldn't hard-code credentials but pass them via environment variables like above. However, I can live with this.